src/vivarium_conic_sam_comparison/model_specifications/vivarium_conic_sam_comparison.in

```diff
@@ -49,6 +49,9 @@ components:
             - InterventionEffect("TF_SAM", 'risk_factor.child_stunting.exposure')
             - NeonatalPreterm()
 
+    vivarium_public_health.metrics:
```

Comment:
can you move this below the PEM special disease block? see here: https://github.com/ihmeuw/vivarium_public_health/blob/develop/src/vivarium_public_health/metrics/disability.py#L60

pem will be excluded otherwise

Reply:
Done.